.github/workflows/fuzz_build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build Fuzz
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build_fuzz:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.16"
```

Comment:
1.15 is used in the other CI workflow, so use it here too.

Reply:
Done